web/public/js/controllers/accept-session-invite-controller.js

```diff
@@ -13,9 +13,8 @@
       ticketId: ticketId,
       invitedUserId: invitedUserId,
       emailSubject: {
-        'request':  $translate.instant('Your ticket request for'),
```

Comment:
Yup. No need for it anymore. This piece of the subject has been rolled into other properties.